management/src/main/resources/application.yml

```diff
@@ -3,5 +3,24 @@ spring:
     view:
       prefix: /WEB-INF/jsp/
       suffix: .jsp
+  rabbitmq:
+      host: localhost
+      port: 5672
+      username: ${RABBIT_USERNAME}
+      password: ${RABBIT_PASSWORD}
+  cloud:
+    stream:
+      bindings:
+        createProductsChannel:
+          destination: productsExchange
```

Comment:
is the destination a queue name?

Reply:
the destination is the name of the rabbitmq exchange that i will be using